src/Makefile.am

```diff
@@ -18,7 +18,7 @@ else
 PTHREAD_DEF =
 endif
 
-PROTOBUF_VERSION = 32:4:0
+PROTOBUF_VERSION = 3.21.4
```

Comment:
do we need to udpate the update_version.py to hit this line?

Reply:
Yep but needs to happen internally, I have filed a bug and CL will come soon.

Reply:
You can just use `-release ${PACKAGE_VERSION}` at which point only configure.ac needs updating.

Reply:
Where is PACKAGE_VERSION set?

Reply:
Either of the two answers should help:
```
AC_INIT([Protocol Buffers],[3.21.4],[protobuf@googlegroups.com],[protobuf])
                            ^^^^^^
```
or
```
grep -r PACKAGE_VERSION /usr/share/aut*
/usr/share/autoconf/autoconf/general.m4:AC_SUBST([PACKAGE_VERSION],
```
it's implied.